libr/cons/cons.c

```diff
@@ -696,12 +715,25 @@ R_API void r_cons_flush() {
 		r_cons_reset ();
 		return;
 	}
+ 	if (lastMatters ()) {
+		if (!CTX (lastMode)) {
+			// snapshot of the output
+			free (CTX (lastOutput));
+			CTX (lastOutput) = calloc (CTX (buffer_len) + 1, 1);
```

Comment:
`R_NEWS0`?

Reply:
im now using malloc when needed, much faster than before, but still handy to have scr.last to disable it at runtime